### PR TITLE
[FIS-277] Feature - mobile offer header last name to display only the initial

### PR DESCRIPTION
**Pull request by IgorZ07**
UI created for FIS-277

Screen Shot 2019-09-05 at 2 21 22 PM

### CODE REVIEW

File: src/components/mobile-offer-header/MobileOfferHeader.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="mobile-offer-header">
```
Comment (WouterSioen) [inline src/components/mobile-offer-header/MobileOfferHeader.vue]: As in the previous PR, if this is a header, shouldn't we use the `header` html element to be semantically correct?

File: src/components/mobile-offer-header/MobileOfferHeader.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <span>{{ text.toUpperCase() }}</span>
+    <div class="wrapper">
+      <p>{{ userName.charAt(0).toUpperCase() + userName.slice(1) }} &nbsp;</p>
```
Comment (WouterSioen) [inline src/components/mobile-offer-header/MobileOfferHeader.vue]: Will this work as intended if you pass an empty string as username?

File: src/components/mobile-offer-header/MobileOfferHeader.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+};
+</script>
+
+<style lang="scss" scoped>
+@import "../../assets/style/main";
+
+.mobile-offer-header {
+  display: flex;
+  flex-flow: column;
+  font-family: "Montserrat", sans-serif !important;
```
Comment (WouterSioen) [inline src/components/mobile-offer-header/MobileOfferHeader.vue]: Shouldn't this be a css variable? This is returning in multiple components?

File: src/components/mobile-offer-header/MobileOfferHeader.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    font-weight: $bold;
+    color: $charcoal-grey;
+    margin-top: $space-xxs;
+  }
+
+  p {
+    margin: 0;
+  }
+}
+</style>
```
Comment (WouterSioen) [inline src/components/mobile-offer-header/MobileOfferHeader.vue]: Your editor should be setup to add an empty line as end-of-file character. The `.editorconfig` plugin + a `.editorconfig` file should already take care of this.

File: src/components/mobile-offer-header/MobileOfferHeader.vue (hunk 2 above)
Comment (IgorZ07) [reply src/components/mobile-offer-header/MobileOfferHeader.vue]: It will not display anything

**Pull request merged**